src/main/java/teammates/ui/controller/AdminInstructorAccountAddAction.java

```diff
@@ -297,37 +298,21 @@ private String getDemoCourseIdRoot(String instructorEmail){
     private String generateNextDemoCourseId(String instructorEmailOrProposedCourseId, int maximumIdLength){
         final boolean isFirstCourseId = instructorEmailOrProposedCourseId.contains("@");
         if(isFirstCourseId){
-            return trimCourseIdToMaximumLengthIfNecessary(getDemoCourseIdRoot(instructorEmailOrProposedCourseId)
-                    , maximumIdLength);
+            return StringHelper.truncateHead(getDemoCourseIdRoot(instructorEmailOrProposedCourseId),
+                                            maximumIdLength);
```

Comment:
I think it's better to align parameters straightly, just 1 more space. Likewise for below.